db.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+    "log"
+    "labix.org/v2/mgo"
+    "time"
+)
+
+// Save the session here for easy access
+var mongoSession *mgo.Session
+
+// Details we need to connect to the database
+// @TODO: Let's use variables from ENV, like
+// os.Getenv("DB_HOST")
+const (
+    // Host of database to connect to
+    DB_HOST = "127.0.0.1" 
+    // Name of database to connect to
+    DB_NAME = "templates"
+    // Username of database user to connect as
+    DB_USER = ""
+    // Password for database user
+    DB_PASS = ""
+    // MongoDB Collection to use for queries
+    DB_COLL = "templates"
+)
+
+type Template struct {
+    Name string  // Name of the template
+    Template string  // The actual template
```

Comment:
Maybe rename 'Template' to something different from the name of the type?
I think something like 'HTML' might be easier to understand, since it reflects what will be stored in this variable.

The comments on this (and the one above) is also IMO explained through the name of the variable, and is thus not needed.